Description: Disable reference.md "Items_and_attributes_15" doctest, failing on armhf
 This passes on i386 amd64 arm64
 .
 See https://gist.github.com/infinity0/2f5452d4ee9deb2f1bacb8b39c987c9c
 for thread stacktrace from 1.10.0 when it hung instead of failed; eddyb from
 #rustc suggested this might be an LLVM issue.
Author: Ximin Luo <infinity0@debian.org>
Bug: https://github.com/rust-lang/rust/issues/40260
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: rustc.git/src/doc/reference.md
===================================================================
--- rustc.git.orig/src/doc/reference.md
+++ rustc.git/src/doc/reference.md
@@ -1133,7 +1133,7 @@ bodies defined in Rust code _can be call
 in the same way as any other Rust function, except that they have the `extern`
 modifier.
 
-```
+```{.ignore}
 // Declares an extern fn, the ABI defaults to "C"
 extern fn new_i32() -> i32 { 0 }
 
